Python/NotARobot/README.md

```diff
@@ -0,0 +1,20 @@
+# NotARobot
+Captcha verification using flask \
```

Comment:
Why are we using Flask here? 

Rotten Scripts, as the name suggests, is composed of traditional scripts